tutorials/graphql/intro-graphql/tutorial-site/content/graphql-mutations.md

```diff
@@ -106,15 +164,226 @@ mutation($todo: todos_insert_input!){
+
+```graphql
+mutation ($id: Int, $is_completed: Boolean, $title: String) {
+  update_todos(where: {id: {_eq: $id}}, _set: {is_completed: $is_completed, title: $title}) {
+    affected_rows
+    returning {
+      id
+      title
+      is_completed
+    }
+  }
+}
+```
+
+Instead of hardcoding the _id_, *is_completed* and _title_ directly into the mutation, we pass them as variables. 
+
+This way, you can re-use the mutation and update other todos without modifying it. All you need to do is to update the Query Variables.
+
+```graphql
+{
+  "id": 55414,
+  "is_completed": true,
+  "title": "Updated - New Todo"
+}
+```
+
+Running the mutation will return the following response:
+
+```graphql
+{
+  "data": {
+    "update_todos": {
+      "affected_rows": 1,
+      "returning": [
+        {
+          "id": 55414,
+          "title": "Updated - New Todo",
+          "is_completed": true
+        }
+      ]
+    }
+  }
+}
+```
+
+The response illustrates the updated todo. Of course, you can tweak the mutation to return something else. For example, you can only leave `affected_rows` and remove the rest.
+
 <b><a href="https://hasura.io/learn/graphql/graphiql" target="_blank">Try it out in GraphiQL</a></b>
 
-We'll explore more mutations to update or delete data a little later.
-This is a good start to grokking mutations!
+### Delete a todo {#delete-a-todo}
+
+Let's try to delete a note from the database. You can do that with the help of the `delete_todos` mutation.
+
+```graphql
+mutation($id: Int) {
+  delete_todos(where: {id: {_eq: $id}}) {
+    affected_rows
+    returning {
+      id
+      title
+    }
+  }
+}
+```
+
+We pass the id as a variable.
+
+```javascript
+{
+  "id": 55386
+}
+```
+
+After the mutation finishes, you will get the following response:
+
+```javascript
+{
+  "data": {
+    "delete_todos": {
+      "affected_rows": 1,
+      "returning": [
+        {
+          "id": 55386,
+          "title": "New Todo"
+        }
+      ]
+    }
+  }
+}
+```
+
+The mutation returns the number of affected rows and the todo title.
+
+<b><a href="https://hasura.io/learn/graphql/graphiql" target="_blank">Try it out in GraphiQL</a></b>
+
+## Nested Mutation
+
+In GraphQL, you can make multiple mutations in one request with the help of nested mutations. When two objects have a relationship between them, you can perform nested mutations.
+
+Let's say we want to add a new todo and its author in the same request. If these objects have a relationship, you can insert both objects - the todo and user - in the same request.
+
+```graphql
+mutation insertTodos {
+  insert_todos(objects: [{title: "Nested Mutations", user: {data: {name: "Hasura"}}}]) {
+    affected_rows
+    returning {
+      id
+      title
+      user {
+        id
+        name
+      }
+    }
+  }
+}
+```
+
+https://hasura.io/learn/graphql/intro-graphql/graphql-queries/
```

Comment:
What is the context for this link being here?

Reply:
Added it by mistake. Will remove it